polling_stations/apps/data_importers/management/commands/import_guildford.py

```diff
@@ -1,8 +1,70 @@
 from data_importers.management.commands import BaseHalaroseCsvImporter
+from django.contrib.gis.geos import Point
 
 
 class Command(BaseHalaroseCsvImporter):
     council_id = "GRT"
-    addresses_name = "2023-05-04/2023-03-14T12:03:03.221735/Eros_SQL_Output014.csv"
-    stations_name = "2023-05-04/2023-03-14T12:03:03.221735/Eros_SQL_Output014.csv"
-    elections = ["2023-05-04"]
+    addresses_name = "2024-05-02/2024-02-28T11:16:15.797223/Eros_SQL_Output001.csv"
+    stations_name = "2024-05-02/2024-02-28T11:16:15.797223/Eros_SQL_Output001.csv"
+    elections = ["2024-05-02"]
+
+    def station_record_to_dict(self, record):
+        # Lancaster Hall, Send Road, Send, Woking GU23 6ET
+        if self.get_station_hash(record) in [
+            "41-lancaster-hall",
+            "42-lancaster-hall",
+        ]:
+            record = record._replace(pollingstationpostcode="GU23 7ET")
+
+        # Fix from council:
+        # Old station: The Granary, Wanborough, Guildford GU3 2JR
+        # New station: Wanborough Village Hall, Wanborough Hill, Wanborough, Guildford (uprn: 10007084973)
+        if (record.pollingstationnumber, record.pollingstationname) == (
+            "37",
+            "The Granary",
+        ):
+            record = record._replace(
+                pollingstationname="Wanborough Village Hall",
+                pollingstationaddress_1="Wanborough Hill",
+                pollingstationaddress_2="Wanborough",
+                pollingstationaddress_3="Guildford",
+                pollingvenueuprn="10007084973",
+                pollingstationpostcode="",
+            )
```

Comment:
How you've done this is totally fine, but I thought it might be a good place to explain a neat python feature. 
When you're passing a bunch of keyword arguments (or `kwargs` - though that sounds like knock off lord of the rings to me...) to a function or method, you can pass it as a dictionary using this syntax: `my_function(**my_dictionary)`. You can also pass a list of positional arguments as `my_function(*my_list)`

This means you could define a dictionary at the top of the file:

```python
# All caps because of its global scope - i.e. defined outside of a `class` or `def` block.
WANBOROUGH_VILLAGE_HALL = {
                "pollingstationname": "Wanborough Village Hall",
                "pollingstationaddress_1": "Wanborough Hill",
                "pollingstationaddress_2": "Wanborough",
                "pollingstationaddress_3": "Guildford",
                "pollingvenueuprn": "10007084973",
                "pollingstationpostcode": ""
}
```

Then in your `x_to_dict` methods you would do:

```python
if (record.pollingstationnumber, record.pollingstationname) == (
            "37",
            "The Granary",
        ):
            record = record._replace(**WANBOROUGH_VILLAGE_HALL)
```

Obviously there are other ways you could do Don't Repeat Yourself - e.g. a function to apply the fix.  And as I said in this case it doesn't really matter, but thought it might be interesting.

Reply:
That is interesting, thanks! I didn't know about that.